aries-site/src/examples/components/avatar/AvatarSizePreviewExample.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Avatar, Box, Text } from 'grommet';
+
+export const AvatarSizePreviewExample = () => (
+  <Box align="center" height="small" direction="row" gap="xlarge">
+    <AvatarContainer
+      alt="Eric Soderberg image"
+      size="small"
+      src="//s.gravatar.com/avatar/99020cae7ff399a4fbea19c0634f77c3?s=80"
+      title="Small avatar with profile picture"
+    ></AvatarContainer>
+    <AvatarContainer
+      background="green"
+      alt="Avatar containing initial letters J and S"
+      title="Medium avatar with initials"
+    >
+      JS
+    </AvatarContainer>
+    <AvatarContainer
+      round="medium"
+      alt="shape logo"
+      size="xlarge"
+      src="/Shape.svg"
+      title="Large avatar with logo"
```

Comment:
<img width="563" alt="Screen Shot 2022-10-19 at 10 42 46 AM" src="https://user-images.githubusercontent.com/12522275/196765871-515cbde5-32ff-4252-947c-8ce943121efb.png">

The logo starts to get squished at certain breakpoints, should the avatar have `flex={false}` always to avoid this?